projects/components/src/datagrid/datagrid.component.html

```diff
@@ -1,3 +1,4 @@
+<h3 *ngIf="this.header">{{ header }}</h3>
```

Comment:
I'm curious as to how this is going to affect the layout when it's set to fill the container height, I'm not sure what would make the `<clr-datagrid>` grow but not the `<h3>`